rust/noosphere-collections/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde = { workspace = true }
 serde_bytes = "0.11"
 byteorder = "^1.4"
 # NOTE: async-once-cell 0.4.0 shipped unstable feature usage
-async-once-cell = "0.4"
+async-once-cell = "0.5"
```

Comment:
We should probably switch to using `tokio::sync::OnceCell`, which is what we use in most other places.